README.md

```diff
@@ -100,6 +100,26 @@ $ docker volume rm MyVolume
 $ /usr/lib/vmware/vmdkops/bin/vmdkops_admin.py volume ls
 ```
 
+## Upgrade Instructions
+
+User's data is not impacted during driver or plugin upgrade.
+vDVS managed plugin is taking care of the control path (management layer) and doesn't deal with data path. So during the upgrade process, only management related operations will be affected and data will be intact. Management operations like attach/detach/create/delete volume using vDVS driver may run into an error for the time being until the upgrade process is finished.
+
+Steps for safe upgrade:
+1. Uninstall VIB (vDVS driver)
```

Comment:
I backed-up and restored the auth-db and everything was working fine.

I performed following steps:
```
1. Created a new vmgroup - vmgroup1 and added vm-1 to it.
2. Created a volume from vm-1.
3. Copied the current auth-db file to different location and then deleted the current auth-db file
4. Upgraded the vib
5. After upgrade, copied back the auth-db file from saved location
6. Did vmgroup ls and I could still the vmgroup1 and vm-1 still belong to the vmgroup1
7. Also vm-1 could only see the volumes belonging to vmgroup1
``